handler/handler.go

```diff
@@ -13,72 +16,147 @@ import (
 type RedisHandler struct {
 	redis.DefaultHandler
 	Start time.Time
+	Sema  *semaphore.Semaphore
 }
 
-// GET
+// GET 2 side
 func (h *RedisHandler) Get(key string) ([]byte, error) {
+	defer h.Sema.Release()
```

Comment:
Ga boleh ngerelease semaphore yang belum diacquire

Reply:
that's right, my bad.